DataFormats/PatCandidates/src/CovarianceParameterization.cc

```diff
@@ -0,0 +1,225 @@
+#include "DataFormats/PatCandidates/interface/CovarianceParameterization.h"
+#include "DataFormats/PatCandidates/interface/liblogintpack.h"
+#include "DataFormats/PatCandidates/interface/libminifloat.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
```

Comment:
I do not see FWCore/ParameterSet in the list of allowed dependencies in https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Restrictions_on_the_package_defi
@Dr15Jones please clarify if this is OK/safe